packages/cyberstorm/src/components/componentStyles/Button.module.css

```diff
@@ -0,0 +1,97 @@
+.button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex: none;
+  gap: var(--gap-medium);
+
+  padding: var(--padding-medium-small) var(--padding-medium);
+  border-radius: var(--size-small);
+
+  background-clip: padding-box, border-box;
```

Comment:
This might be redundant, and can be applied straight to the classes below where they are needed - you can also use `background` shorthand since we want to override bgcolor anyway.

Reply:
A bit redundant yeah. Moved things around, is it ok now?